src/status_im2/common/nickname_drawer/style.cljs

```diff
@@ -0,0 +1,32 @@
+(ns status-im2.common.nickname-drawer.style
+  (:require [quo2.foundations.colors :as colors]))
+
+(defn context-container
+  []
+  {:flex-direction   :row
+   :background-color (colors/theme-colors colors/neutral-10 colors/neutral-80)
+   :border-radius    20
+   :align-items      :center
+   :align-self       :flex-start
+   :padding          4
+   :margin-top       8
+   :margin-left      -4
```

Comment:
it's alright with negative margin?

Reply:
Yup Jamie .. We use it in lots of places